src/styles/main.scss

```diff
@@ -1,7 +1,220 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@mixin onDesktop {
+  @media (min-width: 1020px) {
+    @content;
+  }
```

Comment:
@NgineZZ this file will contain only imports when you're done with the first block of the landing, right?